test/test-changelist.c

```diff
@@ -188,9 +188,12 @@ main(int argc, char **argv)
 
 	/* Initialize a timeout to terminate the test */
 	timeout = evtimer_new(base,timeout_cb,&timeout);
+	if (timeout == NULL)
+		return (1);
```

Comment:
You need to add a cleanup path in this case, since `base` will leak.

Reply:
I'll fix it